### DISCUSSION TITLE
is unofficial installers allowed

### DISCUSSION THREAD

**Discussion by sastofficial** [microsoft/winget-pkgs#38681] [Q&A]
hello if a program doesnt have a installer are you allowed to make a unofficial installer to put it on winget?

(Edit: i dont know if this is a answer or not https://github.com/microsoft/winget-pkgs/discussions/35008)

**Comment by ItzLevvie** [comment]


**Reply by sastofficial** [reply]
another question. if i get permission from the creator of the program to make a installer with inno/advanced installer?

**Reply by sastofficial** [reply]
ok thx for answering
